man/GGIR.Rd

```diff
@@ -152,8 +152,11 @@ GGIR(mode = \Sexpr{format(formals(GGIR::GGIR)[["mode"]])},
 
       \item{acc.metric}{
         Boolean (default = \Sexpr{GGIR::load_params()[["params_general"]][["acc.metric"]]}).
```

Comment:
It is not a boolean but a character argument, correct?

Reply:
Yes! You're right, I've fixed this now.